### PR TITLE
If specified, posts event data to URL upon instance interruption action.

### PR DESCRIPTION
**Pull request by voriol**
If specified, posts event data to URL upon instance interruption action

https://github.com/kubernetes/kops/pull/14562

### CODE REVIEW

**Comment by olemarkus** [comment]
/ok-to-test

Except for tests being outdated, it looks good. Can you please add release notes too?

**Comment by olemarkus** [comment]
This can be cherry picked into 1.26 if we get this in soon.

**Comment by voriol** [comment]
Hi @olemarkus, is all OK or i need to do something with the tests?? Thanks!

**Comment by hakman** [comment]
Try running `hack/update-expected.sh`.

**Comment by voriol** [comment]
hello! Do you think it is interesting to end this pull request?

sorry, I am not able to understand what is failing, I need help to pull this off.

if you prefer I can close it as "unresolved".

thank you

File: upup/models/cloudup/resources/addons/node-termination-handler.aws/k8s-1.11.yaml.template [hunk 1]
```diff
@@ -212,6 +212,8 @@ spec:
               value: "{{ DefaultQueueName }}"
             - name: WORKERS
               value: "10"
+            - name: WEBHOOK_URL
+              value: {{ .WebhookURL }}
```
Comment (hakman) [inline upup/models/cloudup/resources/addons/node-termination-handler.aws/k8s-1.11.yaml.template:216]:
```suggestion
            {{ if .WebhookURL }}
            - name: WEBHOOK_URL
              value: {{ .WebhookURL }}
            {{ end }}
```

File: upup/models/cloudup/resources/addons/node-termination-handler.aws/k8s-1.11.yaml.template [hunk 2]
```diff
@@ -376,6 +378,8 @@ spec:
               value: "false"
             - name: UPTIME_FROM_FILE
               value: "/proc/uptime"
+            - name: WEBHOOK_URL
+              value: {{ .WebhookURL }}
```
Comment (hakman) [inline upup/models/cloudup/resources/addons/node-termination-handler.aws/k8s-1.11.yaml.template:382]:
```suggestion
            {{ if .WebhookURL }}
            - name: WEBHOOK_URL
              value: {{ .WebhookURL }}
            {{ end }}
```

**Review by hakman — CHANGES_REQUESTED** [review]

**Comment by hakman** [comment]
@voriol Sorry for the late review. Please see my suggestions in case you still want to merge this. Thanks!

**Comment by voriol** [comment]
hi @hakman, thanks for your review, better late than never :heart: 

I'll take a look between now and next week

thanks again

**Comment by hakman** [comment]
> hi @hakman, thanks for your review, better late than never ❤️
> 
> I'll take a look between now and next week
> 
> thanks again

Once you have some time, please run `hack/update-expected` and squash the commits and move the release note to 1.28.

**Pull request closed without merging**